### DISCUSSION TITLE
How do I expand the full relative path of a file in file picker?

### DISCUSSION THREAD

**Discussion by theowenyoung** [helix-editor/helix#3724] [Q&A]
My file hierarchy is very long, I can't see the full file path in file picker, how do I expand the full path?

<img width="903" alt="Screen Shot 2022-09-06 at 13 37 01" src="https://user-images.githubusercontent.com/62473795/188555127-2350fae5-5acc-4c4b-873f-5791c4281306.png">

**Comment by David-Else** [comment]
`ctrl-t` toggles the preview off so you can see much more, does that do it?

**Reply by theowenyoung** [reply]
yes, much better, but it is still not enough to show the full path...
